products/samsungmobile.md

```diff
@@ -49,6 +49,36 @@ releases:
     eol: 2028-03-24 # "five years of security updates" (https://news.samsung.com/global/the-samsung-galaxy-a54-5g-and-galaxy-a34-5g-awesome-experiences-for-all)
     link: https://doc.samsungmobile.com/SM-A546U/TMB/doc.html
 
+-   releaseCycle: "Galaxy Tab S9"
+    releaseDate: 2023-08-11
+    eoas: false
+    eol: false
+    link: https://doc.samsungmobile.com/SM-X816B/INS/doc.html
+
+-   releaseCycle: "Galaxy Tab S9+ 5G"
+    releaseDate: 2023-08-11
+    eoas: false
+    eol: false
+    link: https://doc.samsungmobile.com/SM-X816B/INS/doc.html
+
+-   releaseCycle: "Galaxy Tab S9 Ultra"
+    releaseDate: 2023-08-11
+    eoas: false
+    eol: false
+    link: https://doc.samsungmobile.com/SM-X816B/INS/doc.html
+
+-   releaseCycle: "Galaxy Tab S9 FE"
+    releaseDate: 2023-08-11
+    eoas: false
+    eol: false
+    link: https://doc.samsungmobile.com/SM-X816B/INS/doc.html
+
+-   releaseCycle: "Galaxy Tab S9 FE+"
+    releaseDate: 2023-08-11
+    eoas: false
+    eol: false
+    link: https://doc.samsungmobile.com/SM-X816B/INS/doc.html
+    
```

Comment:
Since the link and dates are exactly the same, we should merge this into a single row?

```suggestion    
```

Reply:
@BiNZGi Is there any advantage to listing all the variants? Given that they share the same link and dates.

Reply:
Listing all the variants is better IMO : if dates change for a single variant at list we won't have to revert the merge.

Reply:
See also this similar change for iphone: https://github.com/endoflife-date/endoflife.date/commit/3885b617d6167d57448cb02043d0d9a1092159bd.